graph/telemetry/istio/appender/health_test.go

```diff
@@ -148,6 +151,10 @@ func setupHealthConfig(services []core_v1.Service, deployments []apps_v1.Deploym
 	business.SetKialiControlPlaneCluster(&business.Cluster{
 		Name: business.DefaultClusterID,
 	})
-	businessLayer := business.NewWithBackends(k8s, nil, nil)
+
+	prom := new(prometheustest.PromClientMock)
+	prom.MockNamespaceServicesRequestRates("testNamespace", "0s", time.Unix(0, 0), model.Vector{})
+	prom.MockAllRequestRates("testNamespace", "0s", time.Unix(0, 0), model.Vector{})
+	businessLayer := business.NewWithBackends(k8s, prom, nil)
```

Comment:
Are there any tests for the new functionality of appending health to the graph nodes?